science/cspice/Portfile

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                cspice
+version             N0066
+categories          science
+platforms           darwin
+maintainers         nomaintainer
+universal_variant   no
+
+description         SPICE: An Observation Geometry System for Planetary Science Missions
+
+long_description    NASA's Navigation and Ancillary Information Facility (NAIF) provides \
+                    this toolkit in order to allow (planetary) scientists access\
+                    to all kinds of space exploration related positioning information.
+
+homepage            http://naif.jpl.nasa.gov/naif/
+master_sites        http://naif.jpl.nasa.gov/pub/naif/toolkit/C/MacIntel_OSX_AppleC_64bit/packages/
+distname            cspice
+extract.suffix      .tar.Z
+
+checksums           rmd160  567ea2e00477ed4d00bc127188b07ff668f3f08a \
+                    sha256  f5d48c4b0d558c5d71e8bf6fcdf135b0943210c1ff91f8191dfc447419a6b12e \
+                    size 38611927
+
+post-extract {
+        xinstall -m 0644 -W "${filespath}" example.c naif0011.tls "${worksrcpath}/doc/"
+}
+
+# no configuration
+use_configure no
+
+# set compiler and linker flags
+pre-build {
+        set env(TKCOMPILER) "${configure.cc}"
+        ### better not override, failure is likely
+        #set env(TKCOMPILEOPTIONS) "-m64 -c -ansi -DNON_UNIX_STDIO ${configure.cflags}"
+        set env(TKLINKOPTIONS) "-m64 ${configure.ldflags} -lm"
+}
```

Comment:
why not use `build.env-append` instead (without the `pre-build` block)?

Reply:
I did not want to re-implement the build system.

Reply:
It seems to work without.

Reply:
It seems to work without.